jupyterhub/app.py

```diff
@@ -1185,6 +1185,10 @@ def _log_file_changed(self, change):
         False, help="""Shuts down all user servers on logout"""
     ).tag(config=True)
 
+    server_tokens = Bool(
+        True, help="""Display JupyterHub version information on admin page"""
```

Comment:
I think we can skip this option. This makes sense to do all the time, I think.

Reply:
I agree, all users can ask the version from the API as it is. I'll keep it wrapped up as a separate commit for easier revert. Ideally we can have this report other component versions, too!